ritm_annotation/core/annotation/state.py

```
@@ -0,0 +1,122 @@
+
+    def to_dict(self):
+        """Convert to dictionary for serialization."""
+        return {
+            'image_path': self.image_path,
+            'image_shape': self.image_shape,
+            'current_object_id': self.current_object_id,
+            'objects': [obj.to_dict() for obj in self.objects],
+        }
+
+    @classmethod
+    def from_dict(cls, data: dict):
```

Ruff (0.14.3)

Missing return type annotation for classmethod `from_dict`

(ANN206)